### PR TITLE
Rails 6

### PR DESCRIPTION
**Pull request by OmTheTurtle**

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -13,12 +13,11 @@
...
   # Enable/disable caching. By default caching is disabled.
-  if Rails.root.join('tmp/caching-dev.txt').exist?
-    config.action_controller.perform_caching = true
-
+  # Run rails dev:cache to toggle caching.
+  if Rails.root.join('tmp', 'caching-dev.txt').exist?
     config.cache_store = :memory_store
     config.public_file_server.headers = {
-      'Cache-Control' => 'public, max-age=172800'
+      'Cache-Control' => "public, max-age=#{2.days.to_i}"
```
Comment (kiralybalint) [inline config/environments/development.rb]: :heart_eyes_cat:

**Comment by OmTheTurtle** [comment]
I agree that it would be safer to do an incremental upgrade, but with the scale of this application I don't think it's necessary. I went to the API changes / deprecations and found nothing that impacts this app. I don't really see the reason to do it with separate commits on the same PR, since it'd be merged together anyway.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.5.1
+FROM ruby:2.6.3
```
Comment (falusi94) [inline Dockerfile]: Here I can recommend the same about Ruby version, it's worth it to use the latest patch version.

**Comment by OmTheTurtle** [comment]
I had my thoughts about this and decided to make the upgrade incrementally. I'll close this and create separate PRs.

**Pull request closed without merging**